services/ingestion/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "2.3"
+services:
+    ingestion:
+        build: .
+        ipc: host
+        volumes:
+            - ${INPUT_DIR}:/input/
+        command: "python3 src/pdf_ingestion.py /input 30"
+        environment:
+            - DBCONNECT
```

Comment:
Does this have to get exported by the user before doing a docker-compose up?

Reply:
Yes. It takes it from the environment that the docker-compose up command is run in.